### ISSUE TITLE
[gradle] "clean" task raise a IOException because jreleaser directory cannot be deleted

### ISSUE THREAD

**Issue by joanhermay** [jreleaser/jreleaser#1074]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Have a clean gradle project (A Java project in the example and using Kotlin DSL)
3. Add the JReleaser plugin
4. Configure "jreleaser" section. The "project" section to be specific, in my case (and in the example): author, license, links and inceptionYear.
5. Run `.\gradlew build`
6. Run `.\gradlew clean`

### Expected Behaviour

`clean` task must delete `\build` directory as usual.

### Actual Behaviour

All folders in the `\build` directory are removed (`\build` folder is expected to be deleted at the end), except `\jreleaser` directory and it's content; a `IOException` is raised.

Following, the exception:

```
> Task :clean FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':clean'.
> java.io.IOException: Unable to delete directory 'C:\Users\joanhermay\Desktop\ioexception-in-gradlew-clean-task-due-to-jrelease-folder-cannot-be-deleted\lib\build'
    Failed to delete some children. This might happen because a process has files open or has its working directory set in the target directory.
    - C:\Users\joanhermay\Desktop\ioexception-in-gradlew-clean-task-due-to-jrelease-folder-cannot-be-deleted\lib\build\jreleaser\trace.log
    - C:\Users\joanhermay\Desktop\ioexception-in-gradlew-clean-task-due-to-jrelease-folder-cannot-be-deleted\lib\build\jreleaser
```

**NOTE:** The `\jreleaser` directory can be deleted if gradle daemond is killed.

### Environment Information

- **Operating System**: Windows 10 Pro 22H2 19045.2251
- **JReleaser Version:** 1.3.1
- **JDK Version:** 17.0.5

```
openjdk version "17.0.5" 2022-10-18 LTS
OpenJDK Runtime Environment Microsoft-6841604 (build 17.0.5+8-LTS)
OpenJDK 64-Bit Server VM Microsoft-6841604 (build 17.0.5+8-LTS, mixed mode, sharing)
```

- **Gradle Version:** 7.6
- **Use custom Gradle global properties?:** No
- **Use custom Gradle project properties?:** No

### Example that reproduces the problem

Clone/Fork the repo: [example-repo](https://github.com/joanhermay/ioexception-in-gradlew-clean-task-due-to-jrelease-folder-cannot-be-deleted)

### Some words

- Is my first issue. If a proper issue name is required following some styling, let me know soon.
- I am new to using jreleaser.

**Comment by aalmiray** [comment]
Still getting errors on Windows https://github.com/jreleaser/jreleaser/actions/runs/3583799159/jobs/6029671397

**Comment by aalmiray** [comment]
Released in v1.4.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.4.0
